Tools/cases_generator/generate_cases.py

```diff
@@ -17,7 +18,7 @@
 
 HERE = os.path.dirname(__file__)
 ROOT = os.path.join(HERE, "../..")
-THIS = os.path.relpath(__file__, ROOT)
+THIS = os.path.relpath(__file__, ROOT).replace(os.path.sep, posixpath.sep)
```

Comment:
I assume this is necessary for WIndows?

Maybe this shouldn't be in a PR named "Stop using _Py_OPCODE and _Py_OPARG macros", but I don't see any harm in it.

Reply:
It's only necessary to stop the comments at the top of the file reflecting which OS the script was last run on.

If I don't make this change, I have to manually fix the regenerated files. No big deal either way, but it was less work to fix the generator.